README.md

```diff
@@ -56,17 +56,54 @@ should be on a subdirectory, it shouldn't be here.
 
 ## Errors
 
-* Wrap/QuietWrap/Unwrappable/Unwrap
-* Errors/CompoundError
+### Wrappers
+
+The `Unwrappable` type represents the classic `Unwrap() error` interface implemented
+by `WrappedError`, while the `Errors` interface represents `Errors() []error`.
+
+There are two factories for `Unwrappable`, the standard `"note: error description"` and a
+quiet not including the text of the original error unless unwrapped first.
+
+* `Wrap(err, note)` with a simple string, or with a formatted note `Wrap(err, format, args...)`.
+* And the same for `QuietWrap(err, note)` or `QuietWrap(err, format, args...)`.
+
+The `Unwrap(err error) []error` helper returns a slice of non-nil sub-errors built
+from the following interfaces:
+* `Unwrap() []error`
+* `Errors() []error`
+* `Unwrap() error`
+
+For agreggating multiple errors and the `Unwrap() []error` or `Errors() []error` interfaces
+we have the `CompoundError`.
+
+### Panic and Recover
+
+A `PanicError` is a special wrapper that includes a StackTrace and can wrap anything
+and it's specially useful when combined when used combined with the standard `recover()` as
+
+```go
+defer func() {
+  if err := core.AsRecovered(recover()); err != nil {
+    // ...
+  }
+}()
+```
+
+This will return `nil` if there was a panic, pass-through the error if it implements
+the `Recovered` interface, or wrap anything else in a `PanicError`.
+
+A `Catcher` is another companion of `PanicError` which will calls a function and returns
+its organic `error` or a `PanicError` if it panicked.
+
+And last are `Panic`, `Panicf` and `PanicWrap` variants of the standard `panic()` that will first create a `PanicError` and throw that instead of the raw argument.
```

Comment:
Please try to reformulate this sentence. There is nothing wrong with it but it just sounds like you are talking to someone or doing a presentation, not documenting functions.